pkg/controllers/management/auth/role_template_lifecycle.go

```diff
@@ -44,13 +55,19 @@ func (rtl *roleTemplateLifecycle) Create(obj *v3.RoleTemplate) (runtime.Object,
 	if err := rtl.enqueuePrtbs(obj); err != nil {
 		return nil, err
 	}
+	if err := rtl.enqueueCrtbs(obj); err != nil {
```

Comment:
Should be able to assert here if its project or cluster and enqueue relevant one: `if obj.Context == "project"`